src/shared/widgets/rich-text/rich-text-widget.scss

```diff
@@ -1,14 +1,38 @@
 .customRichTextWidget {
 
-  :global .modal-dialog {
+  :global .ccrte-modal-dialog {
     margin: 0;
 
     .dialog {
+      top: 0;
```

Comment:
In the authoring environment, we can be in a fairly small scrollable iframe, so the top of the frame is often the best we can do.

Reply:
This makes sense.